pkg/templates/kubeadm/v1beta3/kubeadm.go

```diff
@@ -177,17 +178,29 @@ func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, er
 		APIServer: kubeadmv1beta3.APIServer{
 			ControlPlaneComponent: kubeadmv1beta3.ControlPlaneComponent{
 				ExtraArgs: map[string]string{
-					"endpoint-reconciler-type": "lease",
-					"service-node-port-range":  cluster.ClusterNetwork.NodePortRange,
-					"enable-admission-plugins": kubeflags.DefaultAdmissionControllers(kubeSemVer),
+					"enable-admission-plugins":      kubeflags.DefaultAdmissionControllers(kubeSemVer),
+					"endpoint-reconciler-type":      "lease",
+					"kubelet-certificate-authority": "/etc/kubernetes/pki/ca.crt",
+					"profiling":                     "false",
+					"request-timeout":               "300s",
+					"service-node-port-range":       cluster.ClusterNetwork.NodePortRange,
+					"tls-cipher-suites":             safeTLSCiphers(),
 				},
 				ExtraVolumes: []kubeadmv1beta3.HostPathMount{},
 			},
 			CertSANs: certSANS,
 		},
 		ControllerManager: kubeadmv1beta3.ControlPlaneComponent{
```

Comment:
Why don't we set `tls-cipher-suites` for kube-controller-manager and kube-scheduler?

Reply:
Report didn't have any problems about `--tls-cipher-suites` them, only kube-apiserver :shrug:

Reply:
I double checked in KKP and we also set this flag only for kube-apiserver.